src/Components/SearchResults.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from "react";
+import moment from "moment";
+import TableHead from "./TableHead";
+import TableRow from "./TableRow";
+import CustomerProfile from "./CustomerProfile";
+
+function SearchResults({ results, newCustomerInfo }) {
+  const [customerID, setCustomerID] = useState(0);
+  const [allCustomers, setAllCustomers] = useState(results);
+
+  useEffect(() => {
+    if (newCustomerInfo) {
```

Comment:
If you follow my comment in `App.js`, you won't need `useEffect` and the state for `allCustomers` in this component any more

Reply:
I'm not sure if implemented them correctly, but I managed to get ridoff all customers, but not the use effect